### PR TITLE
AUI-2104 Disable some features while user moves a row or a col

### PR DESCRIPTION
**Pull request by fernandosouza**
@mairatma. I could reduce the amount of commits and improve the tests.

### CODE REVIEW

**Comment by mairatma** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-layout/js/aui-layout-builder.js [hunk 1]
```diff
@@ -73,6 +75,38 @@ A.LayoutBuilder = A.Base.create('layout-builder', A.Base, [
         this.get('container').empty();
     },
 
+    _afterLayoutMoveEnd: function() {
```
Comment (mairatma) [inline src/aui-layout/js/aui-layout-builder.js]: This and `_afterLayoutMoveStart` are missing docs

**Comment by mairatma** [comment]
Lgtm, there's just the missing docs problem I commented by inline. With that fixed, we can push this :)

File: src/aui-layout/js/aui-layout-builder.js (hunk 1 above)
Comment (fernandosouza) [inline src/aui-layout/js/aui-layout-builder.js]: Sorry. I forgot.

**Comment by fernandosouza** [comment]
Thank you @mairatma.

**Comment by fernandosouza** [comment]
Done @mairatma

**Comment by mairatma** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**